### PR TITLE
feat(cdk/testing): create a webdriver harness environment

### PR DESCRIPTION
**Pull request by mmalerba**

### CODE REVIEW

**Comment by mmalerba** [comment]
@jelbourn @devversion this is ready for final review to merge the feature branch

File: src/cdk/testing/tests/webdriver-test.bzl [hunk 1]
```diff
@@ -0,0 +1,34 @@
+load("//tools:defaults.bzl", "jasmine_node_test")
+load("@io_bazel_rules_webtesting//web:web.bzl", "web_test")
+load("//tools/server-test:index.bzl", "server_test")
+
+def webdriver_test(name, tags = [], **kwargs):
+    jasmine_node_test(
+        name = "%s_jasmine_test" % name,
+        data = [
+            "@npm//:node_modules",
```
Comment (devversion) [inline src/cdk/testing/tests/webdriver-test.bzl:9]: Bringing in the whole `node_modules` is usually causing extremely large runfile tree's and slower builds. Can we just use fine-grained deps?

File: src/cdk/testing/tests/webdriver.e2e.spec.ts [hunk 2]
```diff
@@ -0,0 +1,135 @@
+import {HarnessLoader, parallel} from '@angular/cdk/testing';
+import {waitForAngularReady, WebDriverHarnessEnvironment} from '@angular/cdk/testing/webdriver';
+import * as webdriver from 'selenium-webdriver';
+import {crossEnvironmentSpecs} from './cross-environment.spec';
+import {MainComponentHarness} from './harnesses/main-component-harness';
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 10_000;  // 10 seconds
```
Comment (devversion) [inline src/cdk/testing/tests/webdriver.e2e.spec.ts:7]: Nit: Add comment on why we increase the timeout?

**Review by devversion — APPROVED** [review]
LGTM.

File: src/cdk/testing/tests/webdriver-test.bzl (hunk 1 above)
Comment (mmalerba) [reply src/cdk/testing/tests/webdriver-test.bzl:9]: good catch, that's just something I did when I was trying to get things working

File: src/cdk/testing/webdriver/webdriver-element.ts [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+import * as webdriver from 'selenium-webdriver';
+import {getWebDriverModifierKeys, webDriverKeyMap} from './webdriver-keys';
+
+/** A `TestElement` implementation for WebDriver. */
+export class WebDriverElement implements TestElement {
+  constructor(
+      readonly element: () => webdriver.WebElement,
+      private _stabilize: () => Promise<void>) {}
+
+  async blur(): Promise<void> {
```
Comment (jelbourn) [inline src/cdk/testing/webdriver/webdriver-element.ts:27]: Do our API docs take the description from `TestElement`, or do we need to duplicate them here?

**Review by jelbourn — APPROVED** [review]
LGTM

File: src/cdk/testing/webdriver/webdriver-element.ts (hunk 3 above)
Comment (mmalerba) [reply src/cdk/testing/webdriver/webdriver-element.ts:27]: It looks like it does not grab the description. This is an issue for the other environments too so I'll fix it in a follow-up

**Pull request merged**